docs_src/design/adr/devops/0010-Release-Artifacts.md

```diff
@@ -0,0 +1,95 @@
+# Release Artifacts
+
+## Status
+
+In Review 06/10/2020
+
+## Context
+
+During the Geneva release of EdgeX Foundry the DevOps WG transformed the CI/CD process with new Jenkins pipeline functionality. After this new functionality was added we also started adding release automation. This new automation is outlined in ADR 0007 Release Automation. However, in ADR 0007 only three release artifact types are outlined. This document is meant to be a living document to try to outline the artifact types for an EdgeX Foundry release and should be modified as that changes.
+
+## Release Artifact Types
+
+### Docker Images
+
+*Tied to Code Release?* Yes
+
+Docker images are released for every named release of EdgeX Foundry. During development the community releases images to the `docker.staging` repository in [Nexus](http://nexus3.edgexfoundry.org). At the time of release we promote the last tested image from `docker.staging` to `docker.release`. In addition to that we will publish the docker image on [DockerHub](https://hub.docker.com/orgs/edgexfoundry).
+
+#### Nexus Retention Policy
+
+##### docker.snapshots
+Retention Policy: 90 days since last download
+
+Contains: Docker images that are not expected to be released. This will contain EdgeX that images are built with each merge and should be available to test things against a specific commit. In addition to this we also build images to optimize the builds in the CI infrastructure.
```

Comment:
**suggest**: These images are built for each project git merge (edgex-go, app services, device services) and are made available to developers can test images that correlate to individual commits. These images are also used to optimize builds in the CI infrastructure.

Reply:
@tonyespy The images for edgex-go, app-services, device-services go into **staging** on each git merge to master. So we would probably move this below.

Reply:
s/can test/so that they can test

For for my education, @lranjbar how do these images help optimise builds in CI infra?

Reply:
At the time this was written images were built with a git commit are currently pushed to both docker.staging and docker.snapshots. I will remove those sentences because it only seems to contain the ci-build-images now.

The images built from the repository [edgexfoundry/ci-build-images](https://github.com/edgexfoundry/ci-build-images)are stored in docker.snapshots in the Nexus. These images are made to optimize the builds.

@tonyespy @ernestojeda @mkbhanda